fedramp_low_rev_4/docs/fedramp_low_rev_4_overview.md

```diff
@@ -0,0 +1,28 @@
+To obtain the latest version of the official guide, please download https://www.fedramp.gov/assets/resources/documents/FedRAMP_Security_Controls_Baseline.xlsx.
+
+## Overview
+
+The Federal Risk and Authorization Management Program (FedRAMP) is a US government-wide program that delivers a standard approach to the security assessment, authorization, and continuous monitoring for cloud products and services.
+
+It is important that Cloud Service Providers (CSPs) understand the impact level of their offering(s) and correlated security categorization when developing their authorization strategy.
+Cloud Service Offerings (CSOs) are categorized into one of three impact levels: Low, Moderate, and High; and across three security objectives: Confidentiality, Integrity, and Availability.
+
+Confidentiality: Information access and disclosure includes means for protecting personal privacy and proprietary information.
+
+Integrity: Stored information is sufficiently guarded against modification or destruction.
+
+Availability: Ensuring timely and reliable access to information.
+
+FedRAMP currently authorizes CSOs at the: Low, Moderate, and High impact levels.
+
+### Low Impact Level
+
+Low Impact is most appropriate for CSOs where the loss of confidentiality, integrity, and availability would result in limited adverse effects on an agency’s operations, assets, or individuals. FedRAMP currently has two baselines for systems with Low Impact data: LI-SaaS Baseline and Low Baseline. The LI-SaaS Baseline accounts for Low-Impact SaaS applications that do not store personal identifiable information (PII) beyond that generally required for login capability (i.e. username, password, and email address). Additionally, information on the security controls involved in FedRAMP’s Low Baseline can be found [here](https://www.fedramp.gov/assets/resources/documents/FedRAMP_Low_Security_Controls.xlsx).
```

Comment:
We can use the link in this block (https://www.fedramp.gov/assets/resources/documents/FedRAMP_Low_Security_Controls.xlsx) at the top of the page instead, as this looks like a more targeted document for the Low level (and we can make the same type of change in Moderate). If we do move it, we can remove this last sentence from here, `Additionally, information on the security controls involved in FedRAMP’s Low Baseline can be found [here](https://www.fedramp.gov/assets/resources/documents/FedRAMP_Low_Security_Controls.xlsx).`.

For each level's document, we should only include the relevant `<Level> Impact Level` paragraph. For instance, in this document, we can remove the Moderate and High Impact Level sections.